.gitignore

```diff
@@ -59,14 +59,15 @@ Thumbs.db
 # Example file exceptions
 !.env.example
 
-# Only track the npm 7 workspace root package-lock.json
+# Only track the root pnpm lockfile, ignore other package manager
 yarn.lock
 package-lock.json
-!/package-lock.json
```

Comment:
Ik zou verwachten dat dan ook de root package.json nu weggehaald zou zijn in deze PR, maar die zie ik niet?

Reply:
Die file was al weg